src/main/java/uk/gov/pay/directdebit/payments/services/SandboxService.java

```diff
@@ -10,10 +10,12 @@
 import uk.gov.pay.directdebit.payments.model.DirectDebitPaymentProvider;
 
 import javax.inject.Inject;
+import java.time.LocalDate;
 import java.util.Map;
 
 public class SandboxService implements DirectDebitPaymentProvider {
     private static final Logger LOGGER = PayLoggerFactory.getLogger(SandboxService.class);
+    private static final LocalDate DAYS_UNTIL_COLLECTION = LocalDate.now().plusDays(4);
```

Comment:
Yikes! This means it will always be four days from when the class was instantiated, which may end up being in the past if the application runs for long enough. Make this an `int` and redo the calculation each time it’s used.

Reply:
OMG - refactoring fail. Sorry!